### PR TITLE
Add spanish translation

### PR DESCRIPTION
**Pull request by LoneWolfHT**

### CODE REVIEW

File: init.lua [hunk 1]
```diff
@@ -3,42 +3,42 @@
...
-	"    c. Change teams.",
-	"6. Don't leave the game whilst in a fight",
-	"7. Don't impersonate other community members",
-	"8. Do not share your password with ANYONE.",
-	"9. Avoid controversial topics like politics/religion",
-	"10. Moderator decisions are final.",
+	S("1. No swearing, dating, or other inappriopriate behaviour."),
+	S("2. No spamming, using all-caps, or any other method of chat abuse."),
+	S("3. Don't be a cheater. No hacked clients."),
+	S("4. Don't excessively spawnkill. Some spawn killing to stop players chasing"),
```
Comment (rubenwardy) [inline init.lua:14]: breaking up the sentence like this, into multiple lines, is kinda awkward.

```lua
S("4. Don't excessively spawnkill."), " ",
S("Some spawn killing to stop players chasing the flag bearer is OK, but hogging the flag and repeatedly killing definitely isn't."),
```

Comment (LoneWolfHT) [reply init.lua:14]: Newlines mess up the list 
image

Comment (LoneWolfHT) [reply init.lua:14]: And translations can't handle it

Comment (LoneWolfHT) [reply init.lua:14]: They don't even allow spaces around the = sign so I can't make the translation file more readable

**Pull request merged**